src/Frontend.hs

```diff
@@ -172,15 +172,15 @@ type AulaMain =
 
 aulaMain :: ActionM m => ServerT AulaMain m
 aulaMain =
-       runHandler Page.viewRooms
+       forceChangeInitialPassword (runHandler Page.viewRooms)
```

Comment:
why these two?  i'm particularly confused by the one wrapping the delegation network.  is that needed?

shouldn't we redirect to a dedicated end-point after login, and that end-point redirects to the actual landing page, whichever that is?  that way if we re-visit that page the check won't happen again, only after the next login.